library/src/main/java/com/getstream/sdk/chat/view/MessageInputView.java

```diff
@@ -393,6 +406,29 @@ private boolean isGrantedPermissions() {
             return true;
     }
 
+    private void showPermissionSettingDialog(){
+        String appName = Utils.getApplicationName(getContext());
+        String msg = appName + getContext().getString(R.string.stream_camera_permission_message);
```

Comment:
The dialog shows `camera` permission issue only. If user allowed `camera`, but blocked `storage` the message will still be about `camera`. So we either need to show correct message passing non granted permission or just show generic message about permissions

Reply:
@elevenetc ,exactly. currently the branch was marked as wip. :) work in progress yet.